src/components/importWallet/ImportFromSeed.vue

```diff
@@ -1,9 +1,9 @@
 <template>
-  <v-form  @submit="addWalletWithPhrase">
+  <v-form  @submit="addWallet">
```

Comment:
[The same story.](https://github.com/endpass/endpass-wallet/pull/220/files#diff-1cd495c50881bf1439f68d975eb589f6R2)